lib/services/common/securityServiceOAuth2.js

```diff
@@ -21,6 +21,8 @@
  * For those usages not covered by the GNU Affero General Public License
```

Comment:
I undesrtand one entry (or more if needed) should be added to CHANGE_NEXT_RELEASE file, corrresponding to the changes included in this PR.

Reply:
Fixed with 7a19d2aaa796262b7eaefd113344d885838727f5